src/ansys/dynamicreporting/core/adr.py

```diff
@@ -0,0 +1,250 @@
+import os
+from pathlib import Path
+import re
+import sys
+from typing import Any, Optional, Type
+
+from django.core import management
+from django.http import HttpRequest
+
+from .adr_utils import get_logger
+from .exceptions import (
+    AnsysVersionAbsentError,
+    DatabaseMigrationError,
+    ImproperlyConfiguredError,
+    InvalidAnsysPath,
+    InvalidPath,
+    StaticFilesCollectionError,
+)
+
+
+class ADR:
+    """
+    >> from ansys.dynamicreporting.core import ADR, Table
+    >> opts = { "CEI_NEXUS_DEBUG" : "0", "CEI_NEXUS_SECRET_KEY":"", "CEI_NEXUS_LOCAL_DB_DIR":r"C:\\cygwin64\\home\vrajendr\\ogdocex" }
```

Comment:
LOL, do you want your login name in a public repo?  @viseshrp

Reply:
lol, thanks